react/modules/sample/src/components/BoundaryFilter.js

```
@@ -0,0 +1,773 @@
+    // **Update selected values once at the end**
+    // debugger;
+    setSelectedValuesCodes(updatedSelectedValues.map((item) => item.code))
+    // debugger;
+    setSelectedValues((prev) => {
+      // Extract existing codes from previous state
+      const existingCodes = new Set(prev.map(item => item.code));
+
+      // Filter out any new values that already exist
+      // const uniqueUpdatedValues = updatedSelectedValues.filter(item => !existingCodes.has(item.code));
+      debugger;
+      return [...updatedSelectedValues]; // Append only unique values
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)